src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -192,8 +219,10 @@ void close() {
      * @param primaryAlignmentSelectionStrategy What to do when there are multiple primary alignments, or multiple
      *                                          alignments but none primary, for a read or read pair.
      * @param addMateCigar                      True if we are to add or maintain the mate CIGAR (MC) tag, false if we are to remove or not include.
-     * @param unmapContaminantReads             If true, identify reads having the signature of contamination from a foreign organism (i.e. mostly clipped bases),
+     * @param unmapContaminantReads             If true, identify reads having the signature of cross-species contamination (i.e. mostly clipped bases),
      *                                          and mark them as unmapped.
+     * @param unmappingReadsStrategy            An enum desribing how to deal with reads whose mapping information is being removed (currently this happens due to cross-species
```

Comment:
- typo in describing.
-  "reads whose mapping information is being removed" -> are being removed?